applications/tari_base_node/src/builder.rs

```diff
@@ -337,13 +378,25 @@ fn setup_comms_services<T>(
     id: Arc<NodeIdentity>,
     peers: Vec<Peer>,
     peer_db_path: &str,
+    data_dir: &PathBuf,
     db: BlockchainDatabase<T>,
     mempool: Mempool<T>,
     consensus_manager: ConsensusManager<T>,
+    factories: CryptoFactories,
 ) -> (CommsNode, Arc<ServiceHandles>)
 where
     T: BlockchainBackend + 'static,
 {
+    // sql lite for wallet, create folders for sql lite
+    let db_name_om = format!("{}.sqlite3", id.node_id().short_str());
```

Comment:
what do `om` and `ts` stand for?

Reply:
If there are multiple sqlite files for a single wallet (why?) then the wallet_file config variable should be change to a path,

Reply:
OM = OutputManager
TS = TransactionService

I think the wallet service requires two files for them, if I am not mistaken a sqllite file = database table. So it uses the two tables.

Reply:
Nope, the Sqlite db is a single file and you should pass the same file path for all the Sqlite backends.

Reply:
awesome thanks for the clarification

Reply:
Look at the tests for the Transaction Service to see how its setup with a single file. The transaction service requires the output manager service but they are all using the same SqliteDB path